lib/s3_file_handler.rb

```diff
@@ -1,5 +1,25 @@
-class S3FileUploader
+class S3FileHandler
+  def self.get_csv_file_from_s3(filename)
```

Comment:
This could be renamed to `get_file_from_s3` (there's nothing CSV specific in its implementation).